gpkit/substitution.py

```diff
@@ -179,8 +178,7 @@ def substitution(nomial, substitutions, val=None):
                 sub = VarKey(name=sub, **descr)
                 exps_[i] += HashVector({sub: x})
                 varlocs_[sub].append(i)
```

Comment:
Naw, you don't know what units a string has? Anyway this is all replaced by keydicts.